### PR TITLE
Add message_helpers.py

### LINKED ISSUE #49: Convert "build_compound" into separate helper function module
See https://colab.research.google.com/drive/12xVw8TBCSB30h4Lf0OeQFH1lEreIInd6#scrollTo=wKUh9_Q5duhb&line=20&uniqifier=1

### PR DESCRIPTION
**Pull request by skearnes**
Fixes #49

### CODE REVIEW

**Comment by skearnes** [comment]
Question: should we optionally run the validator at the end of the function?

File: ord_schema/message_helpers.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            compound.mass.CopyFrom(amount_pb)
+        elif isinstance(amount_pb, schema.Moles):
+            compound.moles.CopyFrom(amount_pb)
+        elif isinstance(amount_pb, schema.Volume):
+            compound.volume.CopyFrom(amount_pb)
+        else:
+            raise TypeError(f'unsupported units for amount: {amount_pb}')
+    if role:
+        field = schema.Compound.DESCRIPTOR.fields_by_name['reaction_role']
+        values_dict = field.enum_type.values_by_name
```
Comment (connorcoley) [inline ord_schema/message_helpers.py]: Nice clean way to do this!

**Comment by connorcoley** [comment]
> Question: should we optionally run the validator at the end of the function?

I don't think we need to since we have these independent unit tests

File: ord_schema/message_helpers.py (hunk 1 above)
Comment (skearnes) [reply ord_schema/message_helpers.py]: Thanks!

**Pull request merged**